src/components/ebay-progress-bar-expressive/examples/default.marko

```diff
@@ -0,0 +1 @@
+<ebay-progress-bar-expressive/>
```

Comment:
For all these, lets make sure we spread input.

Just change it to 
```
<ebay-progress-bar-expressive ...input/>
```
That way any changes on storybook will be reflected.

Reply:
Oh yes! Good catch. Thanks!